packages/framework-core/package.json

```diff
@@ -55,6 +55,7 @@
     "faker": "5.1.0",
     "metadata-booster": "0.3.1",
     "mock-jwks": "1.0.0",
+    "nock": "11.8.2",
```

Comment:
why do we need `nock` here?

Reply:
It's a peer dependency of `mock-jwks`: https://github.com/Levino/mock-jwks/blob/main/package.json
(Sadly, the package also brings a lot of dev dependencies like Koa which seem not even be used.)